src/popup/components/MainBalance.vue

```diff
@@ -21,8 +22,12 @@ export default defineComponent({
     protocol: { type: String as PropType<Protocol>, required: true },
   },
   setup(props) {
+    const fractionPosition = {
+      [PROTOCOL_AETERNITY]: 2,
+      [PROTOCOL_BITCOIN]: 8,
+    };
```

Comment:
I strongly advise against storing protocol specific data in global components.

Reply:
This one is resolved by #2249